### ISSUE TITLE
[Accessibility] The mouse tooltip of second TabPage is not correct when setting TabControl, tabpages' tooltip, and another control's tooltip

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4750]
* .NET Core Version: 
.Net 6.0 SDK build: 6.0.100-Preview.4.21180.17
* Have you experienced this same bug with .NET Framework?: 
No, framework is not supported TabPage's keyborad tooltip
* More Info
1. This is a regression issue, found after the fix of PR: https://github.com/dotnet/winforms/pull/4421
2. This issue cannot repro if don't do following step4

**Repro steps:**
1.	Create a Winforms .Net Core app
2.	Add a TabControl, a Tooltip, a Button to form designer 
3.	Focus on TabControl, go to Properties window, set TabControl's ShowToolTips as True, then set TabControl's Tooltip on toolTip1 property as "TabControl", set Tabpages' TooltipText with "TabPage1" and "TabPage2"
4.	Set anther control: button's Tooltip on toolTip1 property as "button1" --- **it is necessary step**
5.	Build and run app

Or open attached application: 
[WinFormsApp12.zip](https://github.com/dotnet/winforms/files/6235818/WinFormsApp12.zip)

**Actual behavior:**
Keyboard to focus on Button, then Tab to Tabpage1, then mouse-over Tagpage2 to show mouse tooltip, Tabpage2 display is not correct: Tabpage2 tooltip shows TabControl's
![image](https://user-images.githubusercontent.com/26474449/113130115-4359e000-924e-11eb-9a9d-b73abc510377.png)
![TabPageTooltipIssue](https://user-images.githubusercontent.com/26474449/113129964-19082280-924e-11eb-8ef7-997a7a66c67d.gif)

**Expected behavior:**
Tabpage2 shows "TabPage2" tooltip

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @merriemcgaw. I would like to add information about the version of this bug. It is not a #4421 regression. I can reproduce the problem for .NET Core 3.0
![Issue-4750-3 0](https://user-images.githubusercontent.com/23376742/135251030-49289af9-c4a9-472d-982f-a02a25c69008.gif)

Also, if I turn on quirks for displaying tooltips:
```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
   <startup>
      <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
   </startup>
   <runtime>
      <!-- AppContextSwitchOverrides values are in the form of 'key1=true|false;key2=true|false  -->
      <!-- Please note that disabling Switch.UseLegacyAccessibilityFeatures, Switch.UseLegacyAccessibilityFeatures.2 and Switch.UseLegacyAccessibilityFeatures.3 is required to disable Switch.System.Windows.Forms.UseLegacyToolTipDisplay -->
      <AppContextSwitchOverrides value="Switch.UseLegacyAccessibilityFeatures=false;Switch.UseLegacyAccessibilityFeatures.2=false;Switch.UseLegacyAccessibilityFeatures.3=false;Switch.System.Windows.Forms.UseLegacyToolTipDisplay=false"/>
   </runtime>
</configuration>
```
then this bug will also be reproducible for .NET Framework 4.8
![Issue-4750-4 8 1](https://user-images.githubusercontent.com/23376742/135251065-44d480b4-a865-4ee7-8736-732eeb4fe430.gif)
8

**Comment by SergeySmirnov-Akvelon** [comment]
This did not work correctly in .NET Core 5.0 and earlier:
![Issue-4750-beforefix](https://user-images.githubusercontent.com/23376742/135410676-4471e890-256d-432f-beb4-4d8a723981fd.gif)

It was fixed in the #4421 for .NET Core 6.0:
![Issue-4750-afterfix](https://user-images.githubusercontent.com/23376742/135410713-6675269c-5f18-4c07-b868-845420758e4c.gif)

**Comment by VinoWang01** [comment]
Verified with .NET SDK 7.0.100-alpha.1.22069.1 build, this issue was fixed now:
![Fixed](https://user-images.githubusercontent.com/90816462/150485225-64deb859-0872-47d6-9b8f-6c5c2290ac9e.gif)

**Comment by VinoWang01** [comment]
Verified with .NET SDK 7.0.100-preview.1.22109.10 build, this issue is fixed, the result is same as above GIF shown.
